### PR TITLE
Add clear event command

### PR DESCRIPTION
**Pull request by jhqiu21**

### CODE REVIEW

**Review by fusin02 — COMMENTED** [review]
Looks good overall, just a minor suggestion to standardise Event

File: src/main/java/seedu/address/logic/commands/ClearEventsCommand.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Person;
+
+/**
+ * Clears all events of the address book
+ */
+public class ClearEventsCommand extends ClearCommand {
+
+    public static final String COMMAND_FIELD = "events";
```
[the remark this answers is not in the document]
Comment (jhqiu21) [reply src/main/java/seedu/address/logic/commands/ClearEventsCommand.java]: Ok, got it! Fixed already.

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 2]
```diff
@@ -1,23 +1,10 @@
 package seedu.address.logic.commands;
 
-import static java.util.Objects.requireNonNull;
-
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-
 /**
- * Clears the address book.
+ * Represents a clear command with hidden internal logic and the ability to be executed.
```
[the remark this answers is not in the document]
Comment (jhqiu21) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: Yes. Yours is much better! Fixed already.

File: src/main/java/seedu/address/logic/commands/ClearEventsCommand.java
[the remark this answers is not in the document]
Comment (jhqiu21) [reply src/main/java/seedu/address/logic/commands/ClearEventsCommand.java:1]: Fixed already.

**Comment by Haris-Irfan** [comment]
Asides from above changes the rest looks good to me!

File: src/main/java/seedu/address/logic/commands/ClearEventsCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Haris-Irfan) [reply src/main/java/seedu/address/logic/commands/ClearEventsCommand.java]: I agree with fusion!

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Haris-Irfan) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: Was a little confused but I see what you guys mean!

[the remark this answers is not in the document]
Comment (jhqiu21) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: We change the javadoc to make it more clear🤩

**Pull request merged**